rest/config.go

```diff
@@ -513,65 +513,40 @@ func (clusterConfig *ClusterConfig) GetCredentials() (string, string, string) {
 	return base.TransformBucketCredentials(clusterConfig.Username, clusterConfig.Password, *clusterConfig.Bucket)
 }
 
-// Reads a ServerConfig from raw data
-func ReadServerConfigFromData(runMode SyncGatewayRunMode, data []byte) (*ServerConfig, error) {
+// LoadServerConfig loads a ServerConfig from either a JSON file or from a URL
+func LoadServerConfig(runMode SyncGatewayRunMode, path string) (config *ServerConfig, err error) {
+	var data io.ReadCloser
```

Comment:
I think calling this dataReader or dataReadCloser will make the code easier to follow.